zrml/prediction-markets/src/tests.rs

```diff
@@ -337,15 +360,80 @@ fn create_market_fails_on_max_oracle_duration() {
                 MarketDisputeMechanism::SimpleDisputes,
                 ScoringRule::CPMM,
             ),
-            crate::Error::<Runtime>::OracleDurationGreaterThanMaxOracleDuration
+            Error::<Runtime>::OracleDurationGreaterThanMaxOracleDuration
         );
     });
 }
 
+#[cfg(feature = "parachain")]
 #[test]
-fn admin_destroy_market_correctly_slashes_permissionless_market_active() {
+fn create_market_with_foreign_assets() {
     ExtBuilder::default().build().execute_with(|| {
-        simple_create_categorical_market(MarketCreation::Permissionless, 0..2, ScoringRule::CPMM);
+        let deadlines = Deadlines {
+            grace_period: <Runtime as crate::Config>::MaxGracePeriod::get(),
+            oracle_duration: <Runtime as crate::Config>::MaxOracleDuration::get(),
+            dispute_duration: <Runtime as crate::Config>::MaxDisputeDuration::get(),
+        };
+
+        // As per Mock asset_registry genesis ForeignAsset(420) has allow_as_base_asset set to false.
```

Comment:
It could be useful here to have a method, which actually fills in the data to the mock asset registry, so that these comments are not required anymore. Just say: `I think it's fine`, if you don't like to change it here. I am okay with this too.

Reply:
Its added in GenesisConfig, similar to how we add default balances to account, otherwise we need to mint balances in very test before we do anything. So I think this fine as comment will guide reader to mock registry.